scapy/layers/tls/session.py

```diff
@@ -566,10 +566,9 @@ def __repr__(self):
                 if len(sid) > 12:
                     sid = sid[:11] + "..."
                 res.append((src, dst, sid))
-        colwidth = map(lambda x: max(map(lambda y: len(y), x)),
-                       apply(zip, res))
-        fmt = "  ".join(map(lambda x: "%%-%ds"%x, colwidth))
-        return "\n".join(map(lambda x: fmt % x, res))
+        colwidth = [max([len(y) for y in x]) for x in apply(zip, res)]
```

Comment:
`max()` can use a generator and save some lists here. Also `colwidth` can be a generator:
```Python
    colwidth = (max(len(y) for y in x) for x in apply(zip, res))
    fmt = "  ".join("%%-%ds"%x for x in colwidth)
    return "\n".join(fmt % x for x in res)